api/src/opentrons/protocol_runner/json_translator.py

```diff
@@ -83,6 +84,32 @@ def _translate_pipette_command(
     return translated_obj
 
 
+def _translate_liquid_command(
+    protocol: ProtocolSchemaV6, command: protocol_schema_v6.Command
+) -> pe_commands.LoadLiquidCreate:
+    liquidId = command.params.liquidId
+    labwareId = command.params.labwareId
+    volumeByWell = command.params.volumeByWell
+    # v6 data model supports all commands and therefor most props are optional.
+    # load liquid command must contain liquidId, labwareId and volumeByWell.
+    assert liquidId is not None
+    assert labwareId is not None
+    assert volumeByWell is not None
+
+    liquid = protocol.liquids[liquidId]  # type: ignore[index]
```

Comment:
We can avoid this `# type: ignore[index]` with:

```suggestion
    assert protocol.liquids is not None
    liquid = protocol.liquids[liquidId]
```